.github/workflows/cargo.yml

```diff
@@ -44,5 +44,9 @@ jobs:
       run: cargo build --all
     - name: Cargo test
       run: cargo test --all
+    - name: Cargo build with static feature
```

Comment:
Should we make these parallel jobs instead? What's the effect of rebuilding with a feature. Will Rust clear out the previous output?

Reply:
I have little idea about Github action. So, I cannot rewrite cargo.yaml to make these parallel.
Could you assist me?
By the way, rebuilding with static feature is ok. msquic will be rebuilt.

Reply:
Thank you for your suggestion. I've made it separate job.